funcframework/framework.go

```diff
@@ -102,14 +102,15 @@ func RegisterCloudEventFunctionContext(ctx context.Context, path string, fn func
 
 // Start serves an HTTP server with registered function(s).
 func Start(port string) error {
-	server, err := initServer()
+	server, err := InitServer()
 	if err != nil {
 		return err
 	}
 	return http.ListenAndServe(":"+port, server)
 }
 
-func initServer() (*http.ServeMux, error) {
+// InitServer constructs an HTTP ServeMux that serves the configured function target.
+func InitServer() (*http.ServeMux, error) {
```

Comment:
Could we name this `NewServer()` instead? That's a little closer to the standard `http.NewServeMux` whereas Init is a little ambiguous to whether or not the Server was started.

Reply:
Ah, sorry. I missed that bit of your original comment suggesting to rename it.

I updated now to name it `NewServeMux`, to make it clear that it returns an `http.ServeMux`, not an `http.Server`.